content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +46,407 @@ permissions:
 }
 ```
 
-> Note: The `route53:ListHostedZonesByName` statement can be removed if you
+> ℹ️ The `route53:ListHostedZonesByName` statement can be removed if you
```

Comment:
@wallrj this was confusing as `hostedZoneID` is not obvious as an optional field of the clusterIssuer.

I feel an example here would show it as the code only skips this API call if the field is specified https://github.com/cert-manager/cert-manager/blob/master/pkg/issuer/acme/dns/route53/route53.go#L312-L326